Test-Source/Find-DockerSocket.Tests.ps1

```diff
@@ -0,0 +1,49 @@
+Import-Module -Force $PSScriptRoot/../Source/Docker.Build.psm1
+Import-Module -Global -Force $PSScriptRoot/Docker.Build.Tests.psm1
+
+InModuleScope $Global:ModuleName {
```

Comment:
Guess this would allow us to get rid of all the dot sourcing off private functions?

Reply:
Yes it would!